app/javascript/lib/Entities/Board/Province.js

```diff
@@ -0,0 +1,58 @@
+import Entity from "../Entity.js"
+
+export default class Province extends Entity {
```

Comment:
Only a partial refactor, not meant to be the complete model just yet

Currently the `Province` entity is only created for evaluation purposes and immediately discarded afterward.

Once enough areas have been updated to support the new system, the goal is to flip all older designs into the newer one, and we'll no longer need to discard these entities after use